src/python/espressomd/utils.pxd

```diff
@@ -92,6 +92,10 @@ cdef extern from "<limits>" namespace "std" nogil:
         T max()
 
 cdef extern from "Vector.hpp":
+    cppclass Vector3dInt:
```

Comment:
Cython has in principal template support, (with the `[]` syntax, live `vector[double]`). Can you please try if you can get this to work without the code duplication with the `Vector3d`?

Reply:
It's not possible, cython does not support non-type template parameters.

Reply:
```c++
template<typename T>
using Vector3 = Vector<3, T>;
```?

Reply:
Oh, I missunderstood... that should be possible